setup.py

```diff
@@ -29,6 +29,7 @@ def load_module(module_path):
         'colorama >= 0.3',
         'lxml >= 4',
         'requests >= 2',
+        'sxsdiff >= 0.3.0'
```

Comment:
```suggestion
        'sxsdiff >= 0.3.0',
```
カンマが抜けてるので、テストに失敗しています。